internal/common/rules/globalrulescache_test.go

```diff
@@ -29,6 +91,10 @@ func registerTestRule() {
 	aux := []float64{2, 3, 3, 2}
 	AuxiliaryVector := mat.NewVecDense(4, aux)
 
-	RegisterNewRule(name, reqVar, *CoreMatrix, *AuxiliaryVector)
+	_, _ = RegisterNewRule(name, reqVar, *CoreMatrix, *AuxiliaryVector)
```

Comment:
I'd definitely at least check this for errors. This can be a `panic`, I think...

Reply:
Agreed and pushed